lightning/mydump/csv_parser.go

```diff
@@ -50,6 +50,8 @@ type CSVParser struct {
 	// fieldIndexes is an index of fields inside recordBuffer.
 	// The i'th field ends at offset fieldIndexes[i] in recordBuffer.
 	fieldIndexes []int
+
+	lastRecord []string
```

Comment:
This field is used nowhere.

Reply:
maybe removed when merging master and resolve conflict before...
revert back: https://github.com/pingcap/tidb-lightning/pull/281/commits/aa83ba885ef47e44bbc2635f42f74b26f87fda4a